models/classes/Security/DataAccess/Repository/ToolKeyChainRepository.php

```diff
@@ -45,6 +45,10 @@ public function findAll(KeyChainQuery $query): KeyChainCollection
             throw new InvalidLtiProviderException('Lti Provider is not found');
         }
 
+        if (empty($ltiProvider->getToolPublicKey())) {
+            return new KeyChainCollection(...[]);
```

Comment:
a bit weird format

Reply:
imo, it is ok 🙃 .

Reply:
It is a variadic operator, so you can typehint arrays as objects. It is PHP feature and we can strict type arrays of special object. Check the `KeyChainCollection` and you will understand.